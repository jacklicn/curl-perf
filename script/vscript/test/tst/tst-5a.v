
var %h

h{ "a" } = "another string"
h{ "b" } = "before time"
h{ "c" } = "c is the best language"


print( h{"a"} .. " : " .. h {"c" } )


var i = 0
var @k = keys(h)

while i < length(k)
  print( "name: " .. k[i] .. " value: " .. h{ k[i] } )
  i = i + 1
end

k = values(h)

i = 0
while i < length(k)
  print( "value: " .. k[i] )
  i = i + 1
end








