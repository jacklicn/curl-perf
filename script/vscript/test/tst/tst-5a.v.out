*** syntax ok ***
*** semantics and code generation ok ***

>another string : c is the best language

>name: a value: another string

>name: c value: c is the best language

>name: b value: before time

>value: another string

>value: c is the best language

>value: before time
